### PR TITLE
Patch2

### PR DESCRIPTION
**Pull request by iammosespaulr**
Patch2 written UI tests using protractor and upgraded the conf file to work with Mozilla Firefox as well
screenshot from 2018-01-10 21-37-42

### CODE REVIEW

**Comment by iammosespaulr** [comment]
screenshot from 2018-01-11 07-14-31
Second patch split tests to increase user readability

File: dronesym-frontend/e2e/app.e2e-spec.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+element(by.id('username')).sendKeys('lollipop');
+element(by.id('password')).sendKeys('lollipop');
+ //Find the submit button and click it
+element(by.css('[class="waves-effect waves-light btn blue darken-3 col s4 offset-s4"]')).click();
+//Check whether it has the same url
+expect(browser.getCurrentUrl()).toEqual('http://localhost:4200/');
+  });
+});
+describe('LOGIN CHECK', function() {
+it('should check the rest of the user interface', function() {
```
Comment (malithsen) [inline dronesym-frontend/e2e/app.e2e-spec.js]: should be able to login with default admin credentials

File: dronesym-frontend/e2e/app.e2e-spec.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+//incorrect username entry and password entry
+element(by.id('username')).sendKeys('lollipop');
+element(by.id('password')).sendKeys('lollipop');
+ //Find the submit button and click it
+element(by.css('[class="waves-effect waves-light btn blue darken-3 col s4 offset-s4"]')).click();
+//Check whether it has the same url
+expect(browser.getCurrentUrl()).toEqual('http://localhost:4200/');
+  });
+});
+describe('LOGIN CHECK', function() {
```
Comment (malithsen) [inline dronesym-frontend/e2e/app.e2e-spec.js]: Why is this in uppercase?

File: dronesym-frontend/e2e/app.e2e-spec.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+element(by.css('user-signup>div:nth-of-type(1)>div:nth-of-type(1)>div:nth-of-type(1)>form>div:nth-of-type(1)>input')).click(); //add user test
+element(by.css('user-signup>div:nth-of-type(1)>div:nth-of-type(1)>div:nth-of-type(1)>form>div:nth-of-type(1)>input')).clear().sendKeys('test');//add username test
+element(by.css('div:nth-of-type(4)>input')).click();
+element(by.css('div:nth-of-type(4)>input')).clear().sendKeys('test'); //add password test
+element(by.css('div:nth-of-type(5)>label')).click();
+element(by.css('div:nth-of-type(5)>input')).click();
+element(by.css('div:nth-of-type(5)>input')).clear().sendKeys('test'); // add confirmation password test
+element(by.css('button')).click();
+});
+it('should check Navigation Buttons i.e  Map and also check map functionality i.e zoom in, out and also drop a test drone', function() {
```
Comment (malithsen) [inline dronesym-frontend/e2e/app.e2e-spec.js]: Split this into multiple test cases. eg: Should change view to Map when clicked on map button. Should zoom in, should zoom out

**Review by malithsen — CHANGES_REQUESTED** [review]

**Comment by iammosespaulr** [comment]
screenshot from 2018-01-11 17-14-16
@malithsen changes made ..
please review and merge

**Comment by iammosespaulr** [comment]
@malithsen  indentations made

**Comment by iammosespaulr** [comment]
@malithsen could you please  review this task https://codein.withgoogle.com/dashboard/task-instances/6291753122398208/

**Pull request merged**